dockers/docker-database/create_single_redis_config.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
this is not very readable. can you create a redis server template and use template to generate the redis server config?

Reply:
using redis command line plus argument instead